docs/github-issue-guide.md

```diff
@@ -1,87 +0,0 @@
-# How to file a GitHub Issue
-
-We use GitHub Issues to log all of our todos and tasks. Here is
-[a good guide](https://guides.github.com/features/issues/) for them if you are
-unfamiliar.
-
-When logging an issue with go-ipfs, it would be useful if you specified the
-below information, if possible. This will help us triage the issues faster.
-Please title your issues with the type. For instance:
-
-- "bug: Cannot add file with `ipfs add`"
```

Comment:
We don't actually do this... Should we start?

Reply:
I am not against it, but unless we do it consistently all it will do is add noise.